### PR TITLE
drop RHEL5, add Debian8/ubuntu16 support

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -27,47 +27,42 @@
...
         "6",
         "7"
       ]
     },
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "6",
-        "7"
+        "8"
```
Comment (ekohl) [inline metadata.json]: If you add 8, why not add 9 as well since that's the current version.

File: metadata.json [hunk 2]
```diff
@@ -27,47 +27,42 @@
...
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "12.04",
-        "14.04"
+        "14.04",
+        "16.04"
```
Comment (ekohl) [inline metadata.json]: Should we also add 18.04? Any known incompatibilites?

File: metadata.json [hunk 3]
```diff
@@ -27,47 +27,42 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5",
```
Comment (ekohl) [inline metadata.json]: RH still supports RHEL 5 so I'd keep this in unless you actually drop compatibility code

File: metadata.json (hunk 1 above)
Comment (bastelfreak) [reply metadata.json]: I added what's currently in travis and does acceptance tests. debian 9 failed so I din't add it. same for ubuntu 18.

File: metadata.json (hunk 3 above)
Comment (bastelfreak) [reply metadata.json]: ah right. this is actually RHEL and not CentOS

**Pull request merged**